hera_qm/xrfi.py

```diff
@@ -1373,7 +1380,15 @@ def chi_sq_pipe(uv, alg='zscore_full_array', modified=False, sig_init=6.0,
 #############################################################################
 
 
-def xrfi_run(ocalfits_file=None, acalfits_file=None, model_file=None, data_file=None, history=None,
+def xrfi_run(ocalfits_file=None, acalfits_file=None, model_file=None, data_file=None,
+             omnical_median_filter=True, abscal_median_filter=True,
+             omnical_chi2_median_filter=True, abscal_chi2_median_filter=True,
+             omnical_mean_filter=True, abscal_mean_filter=True,
+             omnical_chi2_mean_filter=True, abscal_chi2_mean_filter=True,
+             omnical_zscore_filter=True, abscal_zscore_filter=True,
+             omnivis_median_filter=True, data_median_filter=False,
+             omnivis_mean_filter=True, data_mean_filter=True,
+             correlations='both', history=None,
```

Comment:
I'm not a fan of this approach. I think we should consider the autos and crosses as fundamentally different data sources (certainly their statistics are quite different). The only reason weren't not doing a median filter on the data by default is speed. But we could easily do a median filter on the autos and then a mean filters on both autos and crosses (probably separately).

My suggestion would be to add `autos_median_filter=True` and `data_mean_filter=True` as defaults. We could also rename the `data` ones `crosses` or something like that.

Reply:
I agree. We should be able to median filter the autos but only mean filter the crosses. I've changed the structure to allow this, putting autos/crosses on the same level as omnivis, abscal, omnical.